src/librustc/mir/interpret/pointer.rs

```diff
@@ -198,11 +196,7 @@ impl<'tcx, Tag> Pointer<Tag> {
         msg: CheckInAllocMsg,
     ) -> InterpResult<'tcx, ()> {
         if self.offset > allocation_size {
-            err!(PointerOutOfBounds {
-                ptr: self.erase_tag(),
-                msg,
-                allocation_size,
-            })
+            err!(PointerOutOfBounds { ptr: self.erase_tag(),msg,allocation_size })
```

Comment:
```suggestion
            err!(PointerOutOfBounds { ptr: self.erase_tag(), msg, allocation_size })
```
If that makes it too wide, split across multiple lines. Omitting the whitespace is not an option.